enterprise/models.py

```diff
@@ -803,6 +808,9 @@ class PendingEnterpriseCustomerUser(TimeStampedModel):
         enterprise_customer (ForeignKey[:class:`.EnterpriseCustomer`]): enterprise customer
         user_email (:class:`django.db.models.EmailField`): user email
 
+    .. pii:: The user_email field contains PII, but locally deleted via enterprise.signals.handle_user_post_save.
```

Comment:
I think we talked about this model. I wonder if it makes sense to clarify that if the user never converts to a full user that their email address will persist. We should at least clarify in that there is an that deletes this data in edx-platform and add `consumer_api` to `pii_retirement`.